mods/player_api/api.lua

```diff
@@ -178,8 +178,8 @@ function minetest.calculate_knockback(player, ...)
 end
 
 -- Check each player and apply animations
-minetest.register_globalstep(function()
-	for _, player in pairs(minetest.get_connected_players()) do
+function player_api.globalstep()
+	for _, player in ipairs(minetest.get_connected_players()) do
```

Comment:
Why the change to `ipairs`? I'm fine with both, but see no good reason to prefer one over the other when order doesn't matter.

Reply:
I am told 'ipairs' is faster because it can assume the keys will only be numbers, ordered from 1 to the last entry

Reply:
> I am told 'ipairs' is faster because it can assume the keys will only be numbers, ordered from 1 to the last entry

The performance difference is negligible.

Reply:
Not that important, I was just wondering whether there was any specific reasoning behind it.